locations/spiders/noahs_ny_bagels.py

```
@@ -1,78 +1,23 @@
-        address2 = response.xpath('//span[@class="c-address-street-2"]/text()').extract_first() or ""
-
-        properties = {
-            "street_address": merge_address_lines([address1, address2]),
-            "phone": response.xpath('//span[@itemprop="telephone"]/text()').extract_first(),
-            "city": response.xpath('//span[@class="c-address-city"]/text()').extract_first(),
-            "state": response.xpath('//span[@itemprop="addressRegion"]/text()').extract_first(),
-            "postcode": response.xpath('//span[@itemprop="postalCode"]/text()').extract_first(),
-            "country": response.xpath('//abbr[@itemprop="addressCountry"]/text()').extract_first(),
-            "ref": ref,
-            "website": response.url,
-            "lat": float(response.xpath('//meta[@itemprop="latitude"]/@content').extract_first()),
-            "lon": float(response.xpath('//meta[@itemprop="longitude"]/@content').extract_first()),
-            "name": response.xpath('//h1[@id="location-name"]/text()').extract_first(),
-        }
-
-        hours = self.parse_hours(response.xpath('//table[@class="c-location-hours-details"]//tbody/tr'))
-
-        if hours:
-            properties["opening_hours"] = hours
-
-        yield Feature(**properties)
-
-    def parse(self, response):
-        urls = response.xpath('//a[@class="Directory-listLink"]/@href').extract()
-        is_store_list = response.xpath('//section[contains(@class,"LocationList")]').extract()
-
-        if not urls and is_store_list:
-            urls = response.xpath('//a[contains(@class,"Teaser-titleLink")]/@href').extract()
-
-        for url in urls:
-            if re.search(r"us/.{2}/.+/.+", url):
-                yield scrapy.Request(response.urljoin(url), callback=self.parse_store)
-            else:
-                yield scrapy.Request(response.urljoin(url))
+    sitemap_urls = ["https://locations.noahs.com/sitemap1.xml"]
```

We've talked about this years ago. What happens when they add `sitemap2.xml`? We silently lose POIs. Go up to `https://locations.noahs.com/sitemap.xml` or even `https://locations.noahs.com/robots.txt`.